pydux/log_middleware.py

```diff
@@ -7,7 +7,7 @@ def log_middleware(store):
     """log all actions to console as they are dispatched"""
     def wrapper(next_):
         def log_dispatch(action):
-            print 'Dispatch Action:', action
+            print('Dispatch Action:', action)
```

Comment:
`from __future__ ...` or similar solution for py2.x please :)

Reply:
+1 to @usrlocalben: `from __future__ import print_function` would avoid py2 interpreting these arguments as a tuple).